tools/opt/opt.cpp

```diff
@@ -163,6 +163,14 @@ Options (in lexicographical order):)",
                around known issues with some Vulkan drivers for initialize
                variables.)");
   printf(R"(
+  --replace-descriptor-variable-index-access
+               Replaces every access using a variable index to array variable
+               |desc| that has a DescriptorSet and Binding decorations with a
```

Comment:
`|desc|` does not make sense in this context.  It is used below because it a name from something used in the description.  Here is it never referenced again, so no need for a name.

The description might be better if is was:

Replaces accesses to descriptor arrays with a variable index with a switch that has a case for every possible value of the index.

Reply:
done